site/hxapi/viewamendcancel/amend_complex.md

```diff
@@ -0,0 +1,162 @@
+---
+
+---
+
+
+# Amend Booking - complex (with reprice)
+
+[API Docs](/hxapi/) > product:[amend](/hxapi/viewamendcancel/amend_complex)
+
+The booking details that can be changed for a  amend are as follows:
```

Comment:
Maybe it should be "... for a **complex** amend are as follows:..."?

Reply:
good spot! will change that now.